cli/sql.go

```diff
@@ -82,16 +82,23 @@ func processOneLine(db *sql.DB, line string) error {
 	// Format all rows using tabwriter.
 	tw := tabwriter.NewWriter(os.Stdout, 0, 8, 0, '\t', 0)
 	fmt.Fprintf(tw, "%s\n", strings.Join(cols, "\t"))
-	strs := make([]string, len(cols))
```

Comment:
can you add a todo (fine to assign it to me) to add a test for the cmdline interface?